modules/channel-web/src/config.ts

```diff
@@ -82,4 +82,10 @@ export interface Config {
    * @default false
    */
   lazySocket: boolean
+
+  /**
+   * Whether or not the delete conversation button is enabled
+   * @default false
+   */
+  enableConversationDeletion: boolean
```

Comment:
If this is really deleteConversation only then this should be be in webchatConfig (all in front end code) check config.ts in the ui. Should be configurable at webchat initialization check applyConfig function in store.tsx i believe